src/android-config.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "relation": ["delegate_permission/common.handle_all_urls"],
+    "target": {
+      "namespace": "android_app",
+      "package_name": "com.destinyitemmanager.app",
```

Comment:
Will this work for both app and beta?

Reply:
They'd need different configs. Was thinking we would just  have it for stable. If people want beta they can still use it as they do now?

Reply:
Sure. In that case can we only copy this file if the env is 'release'?